README.md

```diff
@@ -317,7 +317,7 @@ This is to support easy local and outside-spacelift operations. Keeping variable
 | <a name="input_space_name"></a> [space\_name](#input\_space\_name) | Place the created stacks in the specified space\_name. Mutually exclusive with space\_id. | `string` | `null` | no |
 | <a name="input_spaces"></a> [spaces](#input\_spaces) | A map of Spacelift Spaces to create | <pre>map(object({<br/>    description      = optional(string, null)<br/>    inherit_entities = optional(bool, false)<br/>    labels           = optional(list(string), null)<br/>    parent_space_id  = optional(string, "root")<br/>  }))</pre> | `{}` | no |
 | <a name="input_terraform_smart_sanitization"></a> [terraform\_smart\_sanitization](#input\_terraform\_smart\_sanitization) | Indicates whether runs on this will use terraform's sensitive value system to sanitize<br/>the outputs of Terraform state and plans in spacelift instead of sanitizing all fields. | `bool` | `false` | no |
-| <a name="input_terraform_version"></a> [terraform\_version](#input\_terraform\_version) | Terraform version to use. | `string` | `"1.7.2"` | no |
+| <a name="input_terraform_version"></a> [terraform\_version](#input\_terraform\_version) | OpenTofu/Terraform version to use. Defaults to the latest available version of the `terraform_workflow_tool`. | `optional(string)` | `null` | no |
```

Comment:
Updating to match our corresponding change below... hopefully this will save you from having to rebuild readme locally! 

```suggestion
| <a name="input_terraform_version"></a> [terraform\_version](#input\_terraform\_version) | OpenTofu/Terraform version to use. Defaults to the latest available version of the `terraform_workflow_tool`. | `string` | `null` | no |
```